modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java

```diff
@@ -135,6 +142,21 @@ public ListTypeEntry updateListTypeEntry(
 		return listTypeEntryPersistence.update(listTypeEntry);
 	}
 
+	private void _validateExternalReferenceCode(
+			String externalReferenceCode, long companyId, long listTypeEntryId)
```

Comment:
I think that we should validate by "listTypeDefinitionId" and not by company. What do you think?